src/app/view/components/listing-data-table/listing-data-table.component.scss

```diff
@@ -14,9 +14,33 @@
 
         .ui-datatable-tablewrapper {
             flex-grow: 1;
+            background: $white;
 
-            table tbody.ui-datatable-data td {
-                cursor: pointer;
+            table tbody.ui-datatable-data tr {
```

Comment:
Why don't just add a wrapper element with a class when there is an icon that you can use to styles? because all this first-child thing is too complex approach. I think we need to make the html better when an icon is present.